### PR TITLE
Configured ESlint Airbnb

### PR DESCRIPTION
**Pull request by kishore0079**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,5 +34,16 @@
...
+  "devDependencies": {
+    "eslint": "^7.14.0",
+    "eslint-config-airbnb": "^18.2.1",
+    "eslint-config-prettier": "^6.15.0",
+    "eslint-plugin-import": "^2.22.1",
+    "eslint-plugin-jsx-a11y": "^6.4.1",
+    "eslint-plugin-prettier": "^3.1.4",
+    "eslint-plugin-react": "^7.21.5",
+    "eslint-plugin-react-hooks": "^4.2.0",
+    "prettier": "^2.2.0"
```
Comment (pranavakkian) [inline package.json:47]: is it required?

**Review by pranavakkian — CHANGES_REQUESTED** [review]
remove package-lock.json

File: .eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        "ecmaVersion": 12,
+        "sourceType": "module"
+    },
+    "plugins": [
+        "react"
+    ],
+    "rules": {"react/jsx-filename-extension": [1, {
+        "extensions": [".js", ".jsx"]}
+        ]
+    }
```
Comment (pranavakkian) [inline .eslintrc.json]:
```suggestion
    "rules": {}
```

**Review by P-1-X-3-L — DISMISSED** [review]

File: .eslintrc.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+            "jsx": true
+        },
+        "ecmaVersion": 12,
+        "sourceType": "module"
+    },
+    "plugins": [
+        "react"
+    ],
+    "rules": {"react/jsx-uses-react": "off",
+        "react/react-in-jsx-scope": "off","react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }]}
```
Comment (pranavakkian) [inline .eslintrc.json]:
```suggestion
    "rules": {
    "react/jsx-uses-react": "off",
    "react/react-in-jsx-scope": "off"
    }
```

File: .eslintrc.json [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+        
+
```
Comment (shivalibaghel7) [inline .eslintrc.json]: Please use proper formatting.

**Review by shivalibaghel7 — CHANGES_REQUESTED** [review]

File: .eslintignore [hunk 5]
```diff
@@ -0,0 +1,2 @@
+reportWebVitals.js
+index.js
```
Comment (pranavakkian) [inline .eslintignore]: you can't ignore this file.

File: src/reportWebVitals.jsx [hunk 6]
```diff
@@ -0,0 +1,21 @@
+/* eslint-disable indent */
```
Comment (pranavakkian) [inline src/reportWebVitals.jsx]: no need

File: src/setupTests.jsx [hunk 7]
```diff
@@ -0,0 +1,2 @@
+/* eslint linebreak-style: ["error", "windows"] */
```
Comment (pranavakkian) [inline src/setupTests.jsx]: no need to disable this, fix it

**Review by pranavakkian — CHANGES_REQUESTED** [review]

File: src/App.test.jsx [hunk 8]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-undef */
```
Comment (pranavakkian) [inline src/App.test.jsx]: remove this

File: src/reportWebVitals.jsx (hunk 6 above)
Comment (pranavakkian) [inline src/reportWebVitals.jsx]: only the file in which we are writing jsx should be with jsx extension

File: src/setupTests.jsx (hunk 7 above)
Comment (pranavakkian) [inline src/setupTests.jsx]: only the file in which we are writing jsx should be with jsx extension

**Review by pranavakkian — CHANGES_REQUESTED** [review]

**Review by shivalibaghel7 — APPROVED** [review]

**Review by pranavakkian — APPROVED** [review]

**Review by P-1-X-3-L — APPROVED** [review]

**Pull request merged**